package.json

```diff
@@ -21,6 +21,7 @@
     "axios": "^1.0.0",
     "bcrypt": "^5.1.1",
     "cors": "^2.8.5",
+    "dotenv": "^16.4.1",
```

Comment:
This is not needed, Nx automatically import the .env variables into your project